### PR TITLE
Lesson 1

### PR DESCRIPTION
**Pull request by AbdBarho**
Introduction to data types and operators

Boolean operators `&& || !` are left out, we introduce them when we talk about conditionals, OR if the lesson is done early.

### CODE REVIEW

File: toc.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+### Table Of Contents
```
Comment (haraldF) [inline toc.md:1]: ok - this means I also have to change my script to add a new lesson. But having a separate toc.md looks a lot better than always changing lesson1.md :) I just wish there was a plugin that could auto-generate the toc based on `#` headers...

Comment (AbdBarho) [reply toc.md:1]: I updated the `newLesson.js` script...
honestly if I had time I would have written the plugin, it doesn't seem so complicated.

File: lesson1.md [hunk 2]
```diff
@@ -1,25 +1,18 @@
...
 
 ### Welcome to JavaScript!
 
-> It does not matter what we cover, but what you discover (Noam Chomsky)
+---
 
+> It does not matter what we cover, but what you discover
+>
+> \- Noam Chomsky
 ---
```
Comment (haraldF) [inline lesson1.md]:
```suggestion
> \- Noam Chomsky

---
```

We need a newline otherwise the formatting is all broken...

File: lesson1.md [hunk 3]
```diff
@@ -182,82 +166,89 @@ Programming involves knowing and doing the following things:
...
+`**` exponentiation
 
 ---
 
-### Data Type: Undefined
+### Quick Quiz
 
-* The basic data type **Undefined** has only one value: `undefined`
-* Example: `undefined`
+Let's say, everyone here with our siblings wants to go to the cinema.
```
Comment (sevtapim) [inline lesson1.md]:
```suggestion
Let's say, everyone here wants to go to the cinema with their siblings.
```

File: lesson1.md [hunk 4]
```diff
@@ -350,143 +312,110 @@ because the operators in JavaScript behave differently depending on the data typ
...
-### Let's practice
-
-Use JavaScript to compute:
+### Data Type: Undefined
 
-* Am I the youngest person in a group of three?
-* Am I younger than at least one person in a group of three?
+*  **Undefined** has only one value: `undefined`
+* it is just `undefined`
+* it means there is not value here
```
Comment (sevtapim) [inline lesson1.md]:
```suggestion
* it means there is no value here
```

File: lesson1.md [hunk 5]
```diff
@@ -350,143 +312,110 @@ because the operators in JavaScript behave differently depending on the data typ
...
-
-```javascript
-true || true   // true
-true || false  // true
-false || true  // true
-false || false // false
-```
 
-Example: To apply to a job, I need to know German OR English.
+We are not going to explain them today, but definitely when we talk about conditionals.
```
Comment (sevtapim) [inline lesson1.md:383]: Any reason for not talking about them in this lesson. Since in the summary part, there is even an example of logical operators.

**Review by sevtapim — CHANGES_REQUESTED** [review]

File: lesson1.md (hunk 5 above)
Comment (AbdBarho) [reply lesson1.md:383]: I don't think we have enough time for them, also, in comparison with everything else, they are confusing.

**Review by sevtapim — APPROVED** [review]

**Review by haraldF — APPROVED** [review]

**Pull request merged**